search-app/test/cmr/search/test/results_handlers/stac_results_handler.clj

```diff
@@ -138,3 +138,117 @@
        :opendap {:title "opendap link"
                  :href "https://f5eil01v.edn.ecs.nasa.gov/opendapurl"
                  :type "text/xml"}})))
+
+(defn- href
+  "Returns the href for the given base query string and page-num"
+  [base-string page-num]
+  (format "http://localhost:3003/granules.stac?%s&page_num=%s" base-string page-num))
+
+(defn- self-link
+  "Returns the self link with the given page-num"
+  [base-string page-num]
+  {:rel "self"
+   :href (href base-string page-num)})
+
+(def ^:private root-link
+  "Defines the root link"
+  {:rel "root"
+   :href "http://localhost:3003/"})
```

Comment:
instead of hardcoding these can we use the values that are set by environment?  OK, well never mind. I see that a lot of our tests have this data hardcoded.

Reply:
this is only for test, there won't be anything other than localhost.